roles/kafka_broker/tasks/rbac.yml

```diff
@@ -41,6 +47,17 @@
       }
     ) }}"
 
+- name: Add Broker's Principal to Controller's Super Users List
+  include_tasks: kraft_rbac_mtls.yml
+  vars:
+    controller_host: "{{ item }}"
+  loop: "{{ groups['kafka_controller'] }}"
+  args:
+    apply:
+      delegate_to: "{{item}}"
```

Comment:
I see two different ways 'delegate_to' is being used in our code.
Does one have any advantage over the other? 
We should just stick to the better of the two, and use it across.

Reply:
when we have "include_tasks" we can't use delegate_to with it directly, we have to pass it this way. When we don't have any include, we use delegate_to directly like in other tasks